src/js/media-container.js

```diff
@@ -19,58 +19,111 @@ template.innerHTML = `
     :host {
       box-sizing: border-box;
       position: relative;
-
-      /* Position controls at the bottom  */
-      display: inline-flex;
-      flex-direction: column-reverse;
+      display: inline-block;
 
       /* Max out at 100% width for smaller screens (< 720px) */
-      max-width: 100%;
+      // max-width: 100%;
       background-color: #000;
     }
 
+    :host(:not([audio])) *[part~=layer] {
```

Comment:
I'm not sure if we want this flex layout to be the default for every layer we've talked about. I see different needs/expectations for the layer called "centered", the layer that has top/bottom/middle chrome in it, and then any less specific layer where you might just expect it to work like any other html element.